base/inc/fields/siteorigin-widget-field-class-loader.class.php

```diff
@@ -29,29 +30,35 @@ static function single(){
 	/**
 	 * Regsiter class prefixes to watch for in this loader
 	 *
-	 * @param $class_prefixes
+	 * @param string|array $class_prefixes
+	 * @param string $group
 	 */
-	public function add_class_prefixes( $class_prefixes ) {
+	public function add_class_prefixes( $class_prefixes, $group = 'core' ) {
```

Comment:
I suspect having a default value will lead to 3rd party devs using the default too which defeats the whole purpose of having the groups.